### ISSUE TITLE
"Remember my card" does not work on cibc.com

### ISSUE THREAD

**Issue by cowwoc** [brave/brave-browser#17225]
In version Brave Version 1.27.108 Chromium: 92.0.4515.107 (Official Build) (64-bit) the "Remember my card" feature in cibc.com does not work.

In Chrome Version 91.0.4472.164 (Official Build) (64-bit), if I provide your card number, password, check "Remember my card" and click "Sign On" then the following screen will prompt you to provide a name for the card.

With Brave, this page is skipped completely. Instead, the subsequent page shows up instead. I tried disabling Brave Shields and Rewards and it did not help. I also allowed the use of Sensors on this page.

**Comment by ryanbr** [comment]
Investigating the issue with a another cibc user @cowwoc, see if its replicable.  Can you try clearing the `cibc.com` cookies via `brave://settings/content/siteDetails?site=https%3A%2F%2Fwww.cibc.com` , then relogin with "Remember my card" enabled.

**Comment by ryanbr** [comment]
Also noticed port scanning from the site, which will cause cpu perf issues. https://github.com/brave/adblock-lists/pull/619  should help

**Comment by cowwoc** [comment]
Okay, I figured it out. To fix "remember my card" you need to:

1. Allow sensor access
2. Allow all trackers & ads

All other defaults are fine...

I would love to know why the website requires "trackers & ads" to function properly but at least I've got a workaround for now.

How do you fix such websites? Do you hard-code the above settings so future users will get the correct settings out of the box?

**Comment by ryanbr** [comment]
If you block ads and trackers, does it re-break it @cowwoc  ?

**Comment by cowwoc** [comment]
@ryanbr I'm not sure I understand the correct. Are you asking if keeping `Trackers & ads blocked (standard)` enabled prevents the feature from working in the first place? Yes.

If I re-enable this feature after the bank card is already saved then it forces me to re-authenticate using 2FA whereas if blocking remains disabled I am not prompted to re-authenticate.

**Comment by ryanbr** [comment]
Okay can you add the following `@@||assets.adobedtm.com^$script,domain=cibc.com` and `@@||google-analytics.com/analytics.js` into `brave://adblock` and then refresh cibc.com page. Does that help?  @cowwoc

**Comment by cowwoc** [comment]
@ryanbr What should I use for the remaining settings? Do I reset them to default?

**Comment by ryanbr** [comment]
Yeah, lets see if this helps.

**Comment by cowwoc** [comment]
Sorry, no. This did not help. It still skips over the screen that prompts the user to enter the name of the card.

**Comment by cowwoc** [comment]
@ryanbr What time of day are you available for video chat? I'm located in the Eastern time zone.

**Comment by ryanbr** [comment]
Sure, 7:00 pm your time (around 11am here). Tomorrow or the following day?  https://savvytime.com/converter/est-to-nzst  

DM me on Twitter via https://twitter.com/fanboynz or Telegram https://t.me/FanboyNZ

@cowwoc

**Comment by cowwoc** [comment]
Version 1.45.127

The delay is now tracked separately by https://github.com/brave/brave-browser/issues/22178.

I am no longer able to get REMEMBER MY CARD working on Brave, no matter what I do.

I tried "Allow sensor access" and "Allow all trackers & ads" which worked in the past but I am no longer getting prompted to assign the card a name. When I log out of the website, it does not remember the card I had just entered.

We're back to square one.
